### ISSUE TITLE
[BUG] Incompatibility with DiceMC Money & Chestshop mod with latest update

### ISSUE THREAD

**Issue by JustAnotherGhostName** [al132mc/alchemistry#307]
Description:

The latest update of alchemistry-1.19.2-2.2.4 on 1.19.2 is incompatible with the DiceMC Money & Signs Shop mod. This was introduced with the latest alchemistry update, and it was working on the previous version (tested on ATM8 version 1.0.10) with no issues. 

This incompatibility results in a mix of server crashes, blocks not being interactable, ect. As the DiceMC Money & Signs Shop is one of the few 1.19.2 mods that adds a server economy & supports sign chest shops with Quarks editable signs, there are not many if any alternatives to this mod for server administrators. 

Steps to preproduce:

1) Install the latest version of alchemistry, along side DiceMC+Money-1.19.2-0.0.2
2) Start up a world on a server
3) Try to interact with any alchemistry block, see an error in console and sometimes crash. 
4) Attempt to break the block, and its unbreakable (ghost block)

Relevant mods:
Dice MC Money Mod ( DiceMC Money & Signs Shop )
https://www.curseforge.com/minecraft/mc-mods/dicemc-money-mod 

Error log generated in console:
https://gist.github.com/JustAnotherGhostName/16c931e2b5043b5c9e5b14d2862a2e60

Edit:
Here is the other mods GitHub repo aswell. I would report the issue there aswell, but they recently closed submitting issues on github and dont offer an alternative that im aware of. 
https://github.com/Caltinor/DiceMC-Money-Mod

**Comment by Dark-Arcana** [comment]
Didn't mean to add that label. Anyway, looks like some issue with the capability being null on an abstract inventory block. If I get time I will look into it.

**Comment by JustAnotherGhostName** [comment]
> Anyway, looks like some issue with the capability being null on an abstract inventory block. If I get time I will look into it.

Thank you very much. Hopefully it is an easy fix with a fast roll out. In the mean time ill have to offer some alternative for the key components for my player base.

**Comment by Dark-Arcana** [comment]
Can you retest your mod with 2.3.0 so we can see if there is still a present issue? Between the two updates, a few things changed with some of the capabilities that might have resolve this issue. Thanks.

**Comment by Geolykt** [comment]
I am fairly certain that I fixed it through the side rework PR in alchemylib.
<https://github.com/SmashingMods/Alchemistry/issues/312> duplicates this issue.

**Comment by Dark-Arcana** [comment]
Yeah your work fixed a lot of issues. So I'm going through and closing the obvious ones.

**Comment by JustAnotherGhostName** [comment]
Will be testing this again in ~30 min after updating the pack (atm8) to 1.0.22 which has 2.3.0.

**Comment by JustAnotherGhostName** [comment]
At first glance, things appear to work. I was able to place down a dissolver and process some items in it without any errors in the console or a crash. Will be unbanning alchemistry on my server and see if any issues pop up in the next week.
